R/rdeephaven/R/client_wrapper.R

```diff
@@ -170,14 +235,27 @@ Client <- R6Class("Client",
         stop(paste0("'table_object' must be a single data frame, tibble, arrow table, or record batch reader. Got an object of class ", table_object_class[[1]], "."))
       }
     },
+
+    #' @description
+    #' Uses an existing ticket to create a new table on the server.
+    #' @param ticket String denoting the ticket to use to create the new table.
+    #' @return TableHandle reference to the new table.
```

Comment:
This is not creating a new table on the server.  The ticket is a key used to retrieve a table from the server.  At least, that is my impression of what it does.

Reply:
I previously sent this to Cristian and he was happy with this doc. I don't totally understand the ticket idea, so I can't say for sure. I will look again.

Reply:
Ask Colin.